### DISCUSSION TITLE
Need maintainers for this projects

### DISCUSSION THREAD

**Discussion by roldanjr** [zidoro/pomatez#211] [General]
First of all, thank you guys @sekwah41 @ls-chloe-schoreisz @AugustasV @hunkjazz @rochadk @WeslleyNasRocha @stevenambs @gillisandrew for your efforts in contributing to this small project. ❤️ 

I would like to apologize for being so inactive nowadays. It's been a while since the last time I visited this project.
Honestly, I don't have much time right now to maintain this project. I would love to hear if one of you is interested in maintaining this for the community. 

Really thank you for your efforts. Keep safe and God bless everyone. 🙏🏼

**Comment by sekwah41** [comment]
I've graduated now since then :D though I have a few projects I'm working on rn so can't actively implement features but I'm happy to make fixes and help review pr's.

If you want to make it easier for maintaining and sorting I can move to try to alter some standards e.g. enforce https://www.conventionalcommits.org/en/v1.0.0/ so we can generate an automatic changelog, version bumping and handle releases using automated pr's.

**Reply by roldanjr** [reply]
Wow, congrats @sekwah41  😊  

That's more than enough bro. Thank you so much 🙆🏼‍♂️

**Reply by sekwah41** [reply]
Id love to see ya around generally if/when ya have the time ^.^

I've flagged the project as watched so I can keep track of any pr's issues and other stuff. I'll look this weekend into setting up the tools.

https://github.com/sekwah41/Naruto-Mod/pull/141 i was just moving to a cleaner release creation tool the other day. This one generates pr's so we can keep track of what features going in and then release them when we feel a decent candidate is ready.

**Reply by roldanjr** [reply]
> If you want to make it easier for maintaining and sorting I can move to try to alter some standards e.g. enforce https://www.conventionalcommits.org/en/v1.0.0/ so we can generate an automatic changelog, version bumping and handle releases using automated pr's.

This is a great idea. I used [semantic-release](https://github.com/semantic-release/semantic-release) for this project.

I sent you an invite to this repository.

**Reply by roldanjr** [reply]
> Id love to see ya around generally if/when ya have the time ^.^

Yes me too! Again thank you so much. 🙇🏼‍♂️  Keep safe.

**Reply by sekwah41** [reply]
> > If you want to make it easier for maintaining and sorting I can move to try to alter some standards e.g. enforce https://www.conventionalcommits.org/en/v1.0.0/ so we can generate an automatic changelog, version bumping and handle releases using automated pr's.
> 
> This is a great idea. I used [semantic-release](https://github.com/semantic-release/semantic-release) for this project.
> 
> I sent you an invite to this repository.

Ah, I saw the merge pr's good ya already following the conventional commits :D What's the best way to get in contact with you getting your opinions on stuff? Just make pr's and discussions and flag you for review or mention you?

**Reply by roldanjr** [reply]
> Ah, I saw the merge pr's good ya already following the conventional commits :D What's the best way to get in contact with you getting your opinions on stuff? Just make pr's and discussions and flag you for review or mention you?

Any will do or you can also contact me if you want via social media you preferred.
I will try to respond as soon as I could :D

**Comment by WeslleyNasRocha** [comment]
I've never maintained any big projects, but i would love to help any way i can

**Reply by sekwah41** [reply]
For now would be good to propose changes and then request the other maintainers for review for feedback and possible tweaks.

Its a great learning experience. 😍 Plus the more the merrier!

**Reply by roldanjr** [reply]
I strongly agree with @sekwah41 ! 
I'll send you an invite @WeslleyNasRocha to this repo. Thank you in advance 🙌🏼

**Comment by gillisandrew** [comment]
Hey I'd be happy to help where I can. I'd like to focus my efforts on developing some testing for the time being as I don't anticipate having time to tackle any new feature development (currently in my studies).

**Reply by roldanjr** [reply]
Thanks @gillisandrew . I will send you an invite for this repo. 🙇🏼‍♂️
